src/monodroid/jni/android-system.cc

```diff
@@ -689,13 +686,13 @@ AndroidSystem::count_override_assemblies (void)
 	return c;
 }
 
-int
+long
```

Comment:
I'm not sure I understand this change.  `int` should be at least a 32-bit value, right?  So why change to `long`?

I'm less fond of `long` because it's more variable in size, which may be a meaningless consideration in this context...

Reply:
`long` isn't more variable in size than `int`, alas. The only real way to avoid confusion is to use the `stdint.h` types (`uint32_t`, `int32_t` etc) but in this case we shouldn't use these - we need to match the external APIs